recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+            cmakeincludepath = ";".join(self.deps_cpp_info.include_paths)
+            self._cmake.configure(
+                source_folder="root-{}".format(version),
+                defs={
+                    # TODO: Remove BUILD_SHARED_LIBS option when hooks issue is resolved
+                    # (see: https://github.com/conan-io/hooks/issues/252)
+                    "BUILD_SHARED_LIBS": "ON",
+                    "fail-on-missing": "ON",
+                    "CMAKE_CXX_STANDARD": self._cmake_cxx_standard,
+                    "gnuinstall": "OFF",
+                    "soversion": "ON",
+                    # Disable builtins and use Conan deps where available
+                    "builtin_pcre": "OFF",
+                    "builtin_lzma": "OFF",
+                    "builtin_zstd": "OFF",
+                    "builtin_lz4": "OFF",
+                    "builtin_glew": "OFF",
+                    "builtin_openssl": "OFF",
+                    "builtin_fftw3": "OFF",
+                    "builtin_cfitsio": "OFF",
+                    "builtin_davix": "OFF",
+                    "builtin_tbb": "OFF",
+                    # Enable builtins where there is no Conan package
+                    "builtin_xxhash": "ON",
+                    "builtin_afterimage": "ON",
+                    "builtin_gsl": "ON",
+                    "builtin_gl2ps": "ON",
+                    "builtin_ftgl": "ON",
+                    "builtin_vdt": "ON",
+                    # No Conan packages available for these dependencies yet
+                    "davix": "OFF",
+                    "pythia6": "OFF",
+                    "pythia8": "OFF",
+                    "mysql": "OFF",
+                    "oracle": "OFF",
+                    "pgsql": "OFF",
+                    "gfal": "OFF",
+                    "tmva-pymva": "OFF",
+                    "xrootd": "OFF",
+                    "pyroot": self._pyrootopt,
+                    # Tell CMake where to look for Conan provided depedencies
+                    "CMAKE_LIBRARY_PATH": cmakelibpath,
+                    "CMAKE_INCLUDE_PATH": cmakeincludepath,
+                    # Configure install directories
+                    # Conan CCI hooks restrict the allowed install directory
+                    # names but ROOT is very picky about where build/runtime
+                    # resources get installed.
+                    # Set install prefix to work around these limitations
+                    # Following: https://github.com/conan-io/conan/issues/3695
+                    "CMAKE_INSTALL_PREFIX": os.sep.join((self.package_folder, "res")),
+                    # Fix some Conan-ROOT CMake variable naming differences
+                    "PNG_PNG_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["libpng"].include_paths
+                    ),
+                    "LIBLZMA_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["xz_utils"].include_paths
```

Comment:
On Windows, backslashes in paths might cause problems (escaping of variables)
```suggestion
                    "PNG_PNG_INCLUDE_DIR": ";".join(
                        self.deps_cpp_info["libpng"].include_paths.replace("\\", "/")
                    ),
                    "LIBLZMA_INCLUDE_DIR": ";".join(
                        self.deps_cpp_info["xz_utils"].include_paths.replace("\\", "/")
```